projects/pgai/pgai/cli.py

```diff
@@ -102,8 +102,15 @@ def get_vectorizer(db_url: str, vectorizer_id: int) -> Vectorizer:
         # The Ollama API doesn't need a key, so `api_key_name` may be unset
         if "api_key_name" in embedding:
             api_key_name = embedding["api_key_name"]
-            api_key = os.getenv(api_key_name, None)
+            cur.execute(
```

Comment:
As a matter of override, wouldn't make sense to follow just the opposite flow? In particular, consider the env var always as top priority and, if not set, then pull from DB.

Reply:
yeh you're right that makes more sense, thanks :D